### DISCUSSION TITLE
How to get a full display of a very long diagnostic message?

### DISCUSSION THREAD

**Discussion by peter-jerry-ye** [helix-editor/helix#4849] [Q&A]
Hi,

I'm having a very long diagnostic message. It appears on the upper right corner, but it doesn't seem to be fully displayed. What is the right way to focus on the diagnostic message and scroll on it? Or is it even possible?

**Comment by eliasp** [comment]
Could you verify, whether that's still a problem in the current release?
For me, very long diagnostics messages are wrapped in Helix 22.12

Alternatively, check out the "Diagnostics Picker" view (`space`, `d`)

**Comment by erasin** [comment]
https://github.com/helix-editor/helix/pull/6059 Still waiting for improvements to virtual text.
